### ISSUE TITLE
prop[tableName] is undefined (for DT_LocalWeaponData)

### ISSUE THREAD

**Issue by steffenagger** [saul/demofile#91]
When running the `demoFile.gameEvents` example (from the readme), we get:

`this.props[tableName]` is undefined for tableName `'DT_LocalWeaponData'` @ https://github.com/saul/demofile/blob/v1.0.0-beta.1/src/entities/networkable.ts#L72

Note: we're testing with recent a GOTV demo from popflash.site.

**Comment by saul** [comment]
Ah - I haven’t actually updated the examples in the readme yet. It doesn’t surprise me that they’re broken - do you have an idea as to how to fix it?

**Comment by steffenagger** [comment]
This is the example we ran:

```js
const demofile = require('demofile');
const fs = require('fs');

fs.readFile('test.dem', function (err, buffer) {
  let demoFile = new demofile.DemoFile();

  demoFile.gameEvents.on('player_death', e => {
    let victim = demoFile.entities.getByUserId(e.userid);
    let victimName = victim ? victim.name : 'unnamed';

    // Attacker may have disconnected so be aware.
    // e.g. attacker could have thrown a grenade, disconnected, then that grenade
    // killed another player.
    let attacker = demoFile.entities.getByUserId(e.attacker);
    let attackerName = attacker ? attacker.name : 'unnamed';

    let headshotText = e.headshot ? ' HS' : '';

    console.log(`${attackerName} [${e.weapon}${headshotText}] ${victimName}`);
  });

  demoFile.parse(buffer);
});
```

I'm guessing that should still be working?

We got the above code working by doing a defined check on the before mentioned line:
```js
return this.props[tableName] && this.props[tableName][varName];
```

**Comment by saul** [comment]
@steffenagger I can't reproduce this on my version of TypeScript (3.1.1). What version are you running?

**Comment by steffenagger** [comment]
Sorry, I didn't update you here! We discovered (friday :grimacing:) that only demos recorded by https://popflash.site throws this, and only for the first round.

(And then weekend happened...)
(I'm running TS 3.1.3, but fairly sure that's not the issue)

**Comment by saul** [comment]
Could you point me to a demo that exhibits this issue?

**Comment by steffenagger** [comment]
Yep here's one: http://s3.popflash.host/popflash-1283929f-ddf6-4c32-9b49-74b3bfc62b12.zip

**Comment by casperstorm** [comment]
Some more information: I am not sure if http://stratkey.com/ is using this module, however they have trouble parsing that specific demo as well. It seems to be the first round which is causing trouble.

I've tried with two different popflash demos now.

**Comment by saul** [comment]
I believe this has been fixed by v1.0.4 - please can you let me know if that's not the case. Thanks
